AnimalSOunds/AnimalSOunds/Cat.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AnimalSOunds
+{
+    public class Cat : Animal
+    {
+        public override string Name => "Cat";

```

Comment:
public override string Name => "Cat - this is not a cat, my PM asked me to change this";

Reply:
sunt in clasa Cat, stiu ca lucrez cu un obiect de tip Cat; in plus, numele ar putea fi "Felix" nu "Cat"